jsonschema-validation.xml

```diff
@@ -415,6 +443,14 @@
                 <t>
                     If absent, it can be considered the same as an empty object.
                 </t>
+                <t>
+                    Validation of this keyword against the instance always succeeds.
```

Comment:
For an object, immediate instance validation is just about what property names are or are not present (as determined by `required`, `minProperties`, `maxProperties`).

`properties` tells us things about certain property names, but does not restrict the set of property names.

Although now that I think about it, technically even `"additionalProperties": false` does not restrict the set of property names, it just ensures that all child values for names not otherwise accounted for fail their validation.

Hmm... that may lead to a better phrasing.  Let me think about it and I'll submit an update.